airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -147,17 +154,26 @@ class IncrementalMessageStream(SlackStream, ABC):
 
     def __init__(self, default_start_date: DateTime, **kwargs):
         self._start_ts = default_start_date.timestamp()
+        self.set_sub_primary_key()
         super().__init__(**kwargs)
 
+    def set_sub_primary_key(self):
```

Comment:
what is the benefit of setting these attributes? I think it's confusing for the developer because it adds a layer of indirection for accessing `primary_key[0]` and `primary_key[1]` without changing those properties/adding anything to them. Would prefer if we didn't do this and just accessed the values directly.

Reply:
The reason for the bug is that contributor who made a typo didn't paste square brackets to get the list item. And if in code we minimize those accessing and add variables for those values. We can avoid those mistakes. 

Yes, this solution adds some complexity to the code.